common/lib/xmodule/xmodule/js/fixtures/matlabinput_problem.html

```diff
@@ -1,43 +1,51 @@
 <div class="problem">
-    <div aria-live="polite">
-        <div>
-  <span>
-    <p>
-    <p></p>
-  </span>
-  <span><section id="textbox_test_matlab_plot1_2_1" class="capa_inputtype cminput">
-    <textarea rows="10" cols="80" name="input_i4x-MITx-2_01x-problem-test_matlab_plot1_2_1" aria-describedby="answer_i4x-MITx-2_01x-problem-test_matlab_plot1_2_1" id="input_i4x-MITx-2_01x-problem-test_matlab_plot1_2_1" data-tabsize="4" data-mode="octave" data-linenums="true" style="display: none;">This is the MATLAB input, whatever that may be.</textarea>
+  <div>
+    <span>
+      <p>
```

Comment:
They were in the fixture before I made updates, it looks like I can remove them though.